package.json

```diff
@@ -5,16 +5,18 @@
   "author": "bvaughn",
   "license": "MIT",
   "repository": "bvaughn/react-virtualized-v10",
-  "main": "dist/index.js",
-  "module": "dist/index.es.js",
-  "jsnext:main": "dist/index.es.js",
```

Comment:
> `jsnext:main` entry point always was rollup specific and was deprecated
(not officially until we have a real standard) a year ago in favor of
universal `module` field.

What is the harm in specifying both fields, as the repo currently does? Why remove this?

Reply:
Because it's useless. `rollup-plugin-node-resolve` by default prefers `module` field. Would be good to not have old stuff in major releases. People look at big projects configurations and follow them.